docs/languages/csharp/expression-trees-explained.md

```diff
@@ -0,0 +1,92 @@
+[Previous -- Overview](expression-trees.md)
+
+# Expression Trees Explained
+Expression Trees are a data structure that defines code. They are based on the same structures
+that a compiler uses to analyze code and generate the compiled output. As you read through this
+tutorial, you will notice quite a bit of similarity between Expression Trees and the types used
+in the Roslyn APIs to build analyzers and CodeFixes. The concepts are similar, and the end result
```

Comment:
"CodeFix" seems like a term that would need defining.  I get the gist of it based on the name, but don't really understand what it is.